client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/ContactsForm/InvestigationAccordion/AccordionContent/TableSearchBar/tableSearchBarStyles.ts

```diff
@@ -0,0 +1,13 @@
+import {makeStyles} from '@material-ui/styles';
+
+const useStyles = makeStyles({
+    serachIcon: {
+        color: '#107F9B'
```

Comment:
i think it is just the primary theme color so it could just be `<Search color='primary'/>`...
if you prefer keeping the class you could refer to the "theme" object recieved in the makeStyles callback

Reply:
yea, this color specifically is from the same color as the exposure search bar from US  https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1247/ and I wanted them to look similar

Reply:
then maybe use those styles in yours? or event better - make the search bar/icon generic?

Reply:
I've just changed both colors to 'primary' as it's awfully similar to '#107F9B' , I'm pretty sure that the specific color was just a result of select-color tool , as its said in the story :
> מטיריאל UI בצבע כחול